link.go

```diff
@@ -284,35 +288,64 @@ func (l *link) setSettleModes(resp *performAttach) error {
 	return nil
 }
 
-func (l *link) mux() {
-	defer l.muxDetach()
-
+func (l *link) doFlow() (ok bool, enableOutgoingTransfers bool) {
 	var (
 		isReceiver = l.receiver != nil
 		isSender   = !isReceiver
 	)
 
+	switch {
+	// enable outgoing transfers case if sender and credits are available
+	case isSender && l.linkCredit > 0:
+		debug(1, "Link Mux isSender: credit: %d, deliveryCount: %d, messages: %d, unsettled: %d", l.linkCredit, l.deliveryCount, len(l.messages), l.countUnsettled())
+		return true, true
+
+	case isReceiver && l.receiver.manualCreditor != nil:
```

Comment:
Would a `debug()` be useful for this `case`?

Reply:
Also, the surrounding `{}` aren't necessary.

Reply:
Yes, very much so. I'll add.

Reply:
Added: debug()
Removed: {}'s
Fixed: (unrelated debug() call in session that was breaking compilation with `tags debug`)